### DISCUSSION TITLE
TEXTJOIN Support

### DISCUSSION THREAD

**Discussion by qcurtemanjc** [handsontable/hyperformula#1164] [Feature requests]
Hello,

Would it be possible to add support for the formula "TEXTJOIN"? It would be incredibly useful for joining cells that may or may not have a value in them. Please see the link below to see an example. 

https://support.google.com/docs/answer/7013992?hl=en

Thank you

**Comment by AMBudnik** [comment]
Hi @qcurtemanjc 

The `TEXTJOIN` is not on the list of the supported functions yet. But you can use the logic of Custom functions to create one. 

Here https://hyperformula.handsontable.com/guide/custom-functions.html is a tutorial on how to create and use a custom function. If you would have any issues while building it or find anything missing in the documentation, please let me know.

Until then, I will mark this issue as a feature request to add `TEXTJOIN` to the list of supported functions and will update you as soon as it gets on the list.

**Comment by sequba** [comment]
@qcurtemanjc  Also, if you want to [contribute](https://hyperformula.handsontable.com/guide/contributing.html) to HyperFormula by adding this function to the core project, feel free to open a pull request ;)

**Comment by sequba** [comment]
We received another request for this function outside GH
